vendor/github.com/heptiolabs/healthcheck/README.md

```diff
@@ -0,0 +1,92 @@
+# healthcheck
+[![Build Status](https://travis-ci.org/heptiolabs/healthcheck.svg?branch=master)](https://travis-ci.org/heptiolabs/healthcheck)
+[![Go Report Card](https://goreportcard.com/badge/github.com/heptiolabs/healthcheck)](https://goreportcard.com/report/github.com/heptiolabs/healthcheck)
+[![GoDoc](https://godoc.org/github.com/heptiolabs/healthcheck?status.svg)](https://godoc.org/github.com/heptiolabs/healthcheck)
+
+Healthcheck is a library for implementing Kubernetes [liveness and readiness](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) probe handlers in your Go application.
+
+## Features
+
+ - Integrates easily with Kubernetes. This library explicitly separates liveness vs. readiness checks instead of lumping everything into a single category of check.
```

Comment:
```suggestion
 - This library integrates easily with Kubernetes because it explicitly separates liveness vs. readiness checks instead of lumping everything into a single category of check.
```

Reply:
Hello @apigirl ,

This is “vendored” code.
I.e.: this is a 3rd party dependency that the `datadog-operator` depends on and that has been copied verbatim inside this repo in order to not depend on the external world.
Such vendored 3rd party component must remain unmodified compared to upstream.
In order to fix the wording of this documentation, a PR should be submitted upstream instead: https://github.com/heptiolabs/healthcheck